config/common-packages.yml

```diff
@@ -2,6 +2,7 @@ type: rpm-ostree
 repos: 
   - https://copr.fedorainfracloud.org/coprs/qoijjj/hardened_malloc/repo/fedora-39/qoijjj-hardened_malloc-fedora-39.repo
```

Comment:
I'll be putting together a secureblue copr account for consistency since it's odd to have our usernames showing up when users run rpm-ostree upgrade. We can keep the codebases in their current locations though.

Reply:
https://copr.fedorainfracloud.org/coprs/secureblue/